app/serializers/health_care_application_serializer.rb

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+class HealthCareApplicationSerializer < ActiveModel::Serializer
+  attributes :id, :state, :form_submission_id, :timestamp
```

Comment:
I don't know how explicit/consistent we want to be in our responses, but for comparison:
A successful synchronous submission returns:
```
{
  success: true,
  formSubmissionId:  'Submission Id From HCA'
  timestamp: 'a Timestamp From HCA'
}
```
and here we return:
```
{
  id: 'ID from ActiveRecord',
  state: 'will initially be "pending", and will update after processing',
  formSubmissionId: 'will initially be null and will update upon successful processing',
  timestamp: 'will initially be null and will update upon successful processing'
}
```

So, basically, the diff is that a response from a request that was processed synchronously contains `success: true` and has no `state` attribute
A response from a request that was processed asynchronously contains `state: 'pending'` and `id:` and has no `success` attribute.

@jbalboni does this work for you, or should we add a explicit flag (e.g. `processedAsync: true (or false)`?

Reply:
That works for me.